server/database/config/dbBuild.sql

```diff
@@ -1,5 +1,5 @@
 BEGIN;
-DROP TABLE IF EXISTS  users,artwork
+DROP TABLE IF EXISTS  users,artwork,cart 
```

Comment:
spaces after commas.
```suggestion
DROP TABLE IF EXISTS  users, artwork, cart 
```